src/store/contributors.ts

```diff
@@ -0,0 +1,27 @@
+import {createSlice, PayloadAction} from '@reduxjs/toolkit';
+
+import {CONTRIBUTORS} from 'constants/store';
+import {Contributors} from 'types';
+
+const initialState = {
+  isLoading: false,
+  items: [] as Contributors,
```

Comment:
kept the key name "items" here to keep it consistent with the generic  [createItemSlice](https://github.com/thenewboston-developers/thenewboston-Frontend/blob/0f36ba9670f3da7d38648658a45020258912975e/src/utils/store.ts#L40C13-L40C29) util we created.